packages/driver/src/cypress/command.ts

```diff
@@ -145,13 +149,4 @@ export class $Command {
   }
 }
 
-// mixin lodash methods
-_.each(['pick'], (method) => {
-  return $Command.prototype[method] = function (...args) {
-    args.unshift(this.attributes)
-
-    return _[method].apply(_, args)
-  }
-})
-
```

Comment:
This code is added 5 years ago and no other lodash function is added to `$Command`. I guess it wouldn't later. So, I moved it inside the class for type.